script.js

```
@@ -0,0 +1,140 @@
+    });
+    
+    input.addEventListener('blur', saveEdit);
+}
+
+// renderTodos関数を更新
+function renderTodos() {
+    const todoList = document.getElementById('todoList');
+    todoList.innerHTML = '';
+    
+    todos.forEach(todo => {
+        const li = document.createElement('li');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'renderTodos'. Consider to delete it or rename it.

'renderTodos' is defined here:


(lint/suspicious/noRedeclare)